gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineFindBugs.groovy

```diff
@@ -30,6 +30,7 @@ class BaselineFindBugs extends AbstractBaselinePlugin {
 
     private static final String DEFAULT_FINDBUGS_VERSION = '3.0.1'
     private static final String DEFAULT_EFFORT = 'max'
+    private static final String ANTIPATTERNS_ARTIFACT = 'com.palantir.antipatterns:antipatterns:1.0-beta.4'
```

Comment:
I just spent a while fiddling with versions (publishToMavenLocalYay), and I can verify that overriding in the way I document works, although it obviously only works for forward versions.